packages/mui-material/src/Chip/Chip.js

```diff
@@ -108,8 +108,8 @@ const ChipRoot = styled('div', {
         pointerEvents: 'none',
       },
       [`& .${chipClasses.avatar}`]: {
-        marginLeft: 5,
-        marginRight: -6,
+        marginInlineStart: 5,
+        marginInlineEnd: -6,
```

Comment:
This feels like a DX regression.

- "left" feels a lot clearer than "inline start" for LTR users.
- "inline start" feels clearer than "left" for RTL users.

Considering that RTL users is about 2% of the user base, it doesn't add much weight to the overall DX experience.